common/grid-layout/layout-en.html

```diff
@@ -0,0 +1,577 @@
+		</div>
+		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+		</div>
+	</header>
+</div>
+
+<h4>Code Sample</h4>
+<pre><code>&lt;div  class="well"&gt;&lt;header&gt;
+	&lt;div class="brand col-xs-5"&gt;
+	&lt;div class="logo"&gt;
+		&lt;!-- Logo content goes here --&gt;
+	&lt;/div&gt;
+	&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
+	&lt;/div&gt;
+&lt;/header&gt;&lt;/div&gt;</code></pre>
+
+<h4>GCWeb Infostripe with .col- Classes</h4>
+<div class="well">
+	<div class="infostripe col-md-8 col-sm-6 col-xs-12">
+		<!-- Content of the infostripe with .col-md-8, .col-sm-6, and .col-xs-12 classes -->
+		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus nibh, finibus ut sagittis lacinia, consectetur sit amet nunc. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Vivamus et lectus at turpis tristique pretium. Aenean vel massa eu neque consequat dictum. Integer elit ante, egestas non eros non, consectetur posuere justo. Nam vel sollicitudin nisl, et imperdiet mauris. Quisque scelerisque tincidunt nibh, sed bibendum mauris laoreet vel. In hac habitasse platea dictumst.</p>
+	</div>
+</div>
+
+<h4>Code Sample</h4>
+<pre><code>&lt;div class="well"&gt;&lt;div class="infostripe <strong>col-md-8 col-sm-6 col-xs-12</strong>"&gt;
+	<!-- Content of the infostripe with .col-md-8, .col-sm-6, and .col-xs-12 classes -->
+	&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed lectus nibh, finibus ut sagittis lacinia, consectetur sit amet nunc. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Vivamus et lectus at turpis tristique pretium. Aenean vel massa eu neque consequat dictum. Integer elit ante, egestas non eros non, consectetur posuere justo. Nam vel sollicitudin nisl, et imperdiet mauris. Quisque scelerisque tincidunt nibh, sed bibendum mauris laoreet vel. In hac habitasse platea dictumst.&lt;/p&gt;
+&lt;/div&gt;&lt;/div&gt;</code></pre>
+
+
+
+
+
+<h4>GCWeb Fluid Container</h4>
+<div class="well">
+<div class="container-fluid">
+	<!-- Content of the fluid container -->
+	<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+</div>
+</div>
+<h4>Code Sample</h4>
+<pre><code>&lt;div class="well"&gt;&lt;div class="container-fluid"&gt;
+	<!-- Content of the fluid container -->
+	&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
+&lt;/div&gt;&lt;/div&gt;</code></pre>
+
+<h4>GCWeb Fixed-width Container</h4>
+<div class="well">
+<div class="container-fixed">
+	<!-- Content of the fixed-width container -->
+	<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+</div>
+</div>
+<h4>Code Sample</h4>
+<pre><code>&lt;div class="well"&gt;&lt;div class="container-fixed"&gt;
+	<!-- Content of the fixed-width container -->
+	&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
+&lt;/div&gt;&lt;/div&gt;</code></pre>
```

Comment:
```suggestion
<p>Working example to come later</p>
<h4>Code Sample</h4>
<pre><code>&lt;main&gt;&lt;div class="container-fixed"&gt;
	<!-- Content of the fixed-width container -->
	&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/p&gt;
&lt;/div&gt;&lt;/main&gt;</code></pre>
```

Feel free to create a working page in the grid-layout, like the container fluid.
See those reference on how we did for the container fluid.
* Working example page: https://github.com/wet-boew/GCWeb/blob/master/templates/content-page/content-fluid-en.md?plain=1
  * Change the filename
  * Change the "altLangPage" page property
  * Replace the "layout" property by the new layout that you are going to created for the container-fixed
  * Reuse the same content
  * The French equivalent are in the same folder
* For the new "container-fixed" layout
  * Clone this layout : https://github.com/wet-boew/GCWeb/blob/master/sites/layouts/fluid.html
  * rename the filename for "container-fixed.html"
  * replace the CSS class "container-fluid" by "container-fixed"
  * Put it in a new folder here: "/common/grid-layout/layouts/"